crates/re_viewer_context/src/annotations.rs

```diff
@@ -94,7 +94,7 @@ impl<'a> ResolvedClassDescription<'a> {
     #[inline]
     pub fn annotation_info(&self) -> ResolvedAnnotationInfo {
         ResolvedAnnotationInfo {
-            class_id: self.class_description.map(|desc| desc.info.id.into()),
+            class_id: self.class_id,
```

Comment:
I suspect there are cases we want this to be something like?
```
self.class_id.or_else(||, self.class_description.map(|desc| desc.info.id.into()))
```

Reply:
I think if we have no class_id but were able to resolve to a class_description something went wrong, no?

Reply:
You're right -- read through the code -- class_description should only be set if class_id is non-None, and the reverse should never be true.

Reply:
let's try to keep this in mind on a future refactor